dataprofiler/profilers/profile_builder.py

```diff
@@ -866,6 +866,31 @@ def report(self, report_options: dict = None) -> dict:
         """
         raise NotImplementedError()
 
+    @classmethod
+    def load_from_dict(cls, data) -> BaseProfiler:
```

Comment:
does this need options (config) as an input arg

Reply:
Yes!

Reply:
good eye @micdavis !